document/core/exec/modules.rst

```diff
@@ -769,7 +769,13 @@ The values :math:`\val_{\F{res}}^m` are returned as the results of the invocatio
 .. math::
    ~\\[-1ex]
    \begin{array}{@{}lcl}
-   \invoke(S, \funcaddr, \val^n) &=& S; \val^n~(\INVOKE~\funcaddr) \\
+   \invoke(S, \funcaddr, \val^n) &=& S; F; \val^n~(\INVOKE~\funcaddr) \\
      &(\iff & S.\SFUNCS[\funcaddr].\FITYPE = [t_1^n] \to [t_2^m] \\
      &\wedge& \val^n = (t_1.\CONST~c)^n) \\
+     &\wedge& \moduleinst = \{~
+       \begin{array}[t]{@{}l@{}}
+       \MITYPES~\epsilon, \MIFUNCS~\epsilon, \MITABLES~\epsilon, \\
+       \MIMEMS~\epsilon, \MIGLOBALS~\epsilon, \MIEXPORTS~\epsilon ~\}
+       \end{array} \\
+     &\wedge& F = \{ \AMODULE~\moduleinst, \ALOCALS~\epsilon \} \\
```

Comment:
The notation section establishes the convention that empty fields can be omitted in a module record, so you can simply define this as F = {module {}, locals \epsilon}.